src/chrome/content/rules/Findlawimages.com.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="Findlaw_Images">
```

Comment:
Please avoid underscores in rulset names.

Reply:
Thanks, I will change that.

Reply:
@roanhawkeye please see also my [previous comment](https://github.com/EFForg/https-everywhere/pull/14866#discussion_r178438665) and the related [chapter in our docs](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts).